application.js

```diff
@@ -84,8 +84,18 @@ function run(cb) {
 
 run(function(err, port) {
   if (err) {
-    console.error(err);
-    process.exit(1);
+    if (err.name === 'MongoError' && _.includes(err.message, 'failed to connect to server')) {
```

Comment:
Can we not just check for the `process.env. FH_MONGODB_CONN_URL` environment variable and don't try and connect until we have it?

This would also work fine for local users.